apps/web/src/features/subjects/pages/SubjectGraphPage.tsx

```diff
@@ -40,16 +44,64 @@ export const SubjectGraphPage = () => {
 
   const lines = useGraphLines({ selectedMeasures });
 
+  const graphRef = useRef<HTMLDivElement>(null);
+
   const handleSelectForm = (id: string) => {
     setInstrumentId(id);
     setSelectedMeasures([]);
   };
 
+  const handleGraphDownload = async () => {
+    if (!graphRef.current) return;
+
+    const selectGraph = graphRef.current;
+
+    const canvas = await html2canvas(selectGraph, {
+      onclone: (_, element) => {
+        const graphDesc = document.createElement('div');
+        graphDesc.innerText = instrument!.details.title + ' of Subject: ' + params.subjectId!.slice(0, 7);
+        graphDesc.className = 'p-2 font-semibold text-center';
+        element.prepend(graphDesc);
+
+        if (selectedMeasures) {
+          const graphMeasure = document.createElement('div');
+          graphMeasure.innerText = 'Form of Measurement: ';
+          for (const i of selectedMeasures) {
```

Comment:
Please rename `i` to something descriptive. It is confusing because `i` is not an index here.